### ISSUE TITLE
Original youtube gets reinstalled.

### ISSUE THREAD

**Issue by recon08** [FriedChickenButt/youtube-webos#20]
On my 2017 WebOS 3, the next day, the original youtube will always be reinstalled automatically, no matter what I do. Dev mode on, extended time but still. Do you have an idea why, and how to persist the installed app?

**Comment by NguyenASang** [comment]
It's weird :))
Can you try to reset your TV and then install this youtube again ?

**Comment by recon08** [comment]
Hm, I can try. I am wondering if it is because its "preinstalled". But whats also weird is that I even tried increasing the appinfo.json's version to 9.9.9 but it still wipes the hacked version.

**Comment by delphir-com** [comment]
maybe that's the reason?

https://webostv.developer.lge.com/develop/app-test/using-devmode-app/#installDevModeApp

```
If you sign in to multiple TVs with one account, only the session of the latest TV that you sign in to is maintained. 

After Developer Mode is disabled, the installed apps that you were using on Developer Mode are uninstalled, and you will be taken to the log-in screen of the Developer Mode app.
```

**Comment by NguyenASang** [comment]
You can try to root your TV :)) Maybe it will fix that

**Comment by delphir-com** [comment]
I`m afraid rooting is not an option anymore with the latest firmwares

**Comment by Informatic** [comment]
Have you uninstalled "original" YouTube TV app first? Apps installed via content store have priority over ones installed in developermode storage.

**Comment by recon08** [comment]
Yeah, I couldnt uninstall it from the LG Content store unfortunately but I uninstalled it from ares cli. I guess it comes preinstalled with my region (central eu)

**Comment by NguyenASang** [comment]
It is possible to uninstall the "original" Youtube on your TV!!
### Instruction
- Turn on your TV, press the "home" button, then press and hold the "left arrow button" to the bottom of the list.
- At the bottom of the list you will see "a tool with a pen logo", press the "OK button" to open
- When it opens, move to the "Original" Youtube app, press the "up arrow button" then press the "Ok button" and it will be uninstalled.

**Comment by tinhnguyen1988** [comment]
I got the same issue

**Comment by vinismarques** [comment]
Is there no other way of keeping the app installed besides manually pressing "Extend" every X hours?

**Comment by NguyenASang** [comment]
@vinismarques You can use this to auto extend the developer time: https://webosapp.club/apps/reset.html

**Comment by vinismarques** [comment]
@NguyenASang Thank you for sharing! I just installed the .ipk directly and it seems to be working fine.

**Comment by mervinsanandres** [comment]
@delphir-com Does this mean that after the developer mode session expires, even if I uninstalled the original youtube app, it will get reinstalled and replace the one I installed using ares?

**Comment by NguyenASang** [comment]
@mervinsanandres No, it will remove "No Ads Youtube" without reinstall "Original Youtube" (at least my TV).

**Comment by abhinavmarwaha** [comment]
> Yeah, I couldnt uninstall it from the LG Content store unfortunately but I uninstalled it from ares cli. I guess it comes preinstalled with my region (central eu)

hey can you give the exact steps? I am not able to uninstall youtube with ares-install.

**Comment by nguyenhuutinh** [comment]
you need to uninstall youtube manually then follow this git to install app > I use Visual Studio Code extension to install
<img width="932" alt="Screen Shot 2022-09-17 at 17 33 46" src="https://user-images.githubusercontent.com/4114159/190852462-173eaff0-be70-462f-ae12-6aa3ad90ce45.png">
